bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/Partners.scala

```diff
@@ -12,7 +12,7 @@ object Partners {
     NgoPartner.BR -> "Brazil",
     NgoPartner.CA -> "Canada",
     NgoPartner.CL -> "Chile",
-    NgoPartner.KR -> "Korea",
+    NgoPartner.KR -> "Republic of Korea",
```

Comment:
Actually the name `Korea` was provided by Andy Adamson as part of the agreements with Korea, I'd say we shouldn't change it. If we do change it we may need to change the upconversion process